apps/fz_http/lib/fz_http/devices/device.ex

```diff
@@ -239,8 +239,8 @@ defmodule FzHttp.Devices.Device do
         else
           add_error(
             changeset,
-            ip_type,
-            "address pool is exhausted. Increase network size or remove some devices."
+            :base,
```

Comment:
Is this a magic value or something? where does this error get displayed?

Reply:
https://github.com/firezone/firezone/blob/fix-ip-exhaustion-error/apps/fz_http/lib/fz_http_web/live/device_live/new_form_component.html.heex#L61

Reply:
It's a Rails convention I'm borrowing to display non-field-specific errors ;-)

Reply:
Cool :)